java/runtime/src/main/java/org/ray/runtime/runner/RunManager.java

```diff
@@ -253,20 +253,21 @@ private void startRaylet() {
     List<String> command = ImmutableList.of(
         // The raylet executable file.
         getTempFile("/raylet").getAbsolutePath(),
-        rayConfig.rayletSocketName,
-        rayConfig.objectStoreSocketName,
-        "0",  // The object manager port.
-        "0",  // The node manager port.
-        rayConfig.nodeIp,
-        rayConfig.getRedisIp(),
-        rayConfig.getRedisPort().toString(),
-        "0", // number of initial workers
-        String.valueOf(maximumStartupConcurrency),
-        ResourceUtil.getResourcesStringFromMap(rayConfig.resources),
-        String.join(",", rayConfig.rayletConfigParameters), // The internal config list.
-        buildPythonWorkerCommand(), // python worker command
-        buildWorkerCommandRaylet(), // java worker command
-        redisPasswordOption
+        String.format("-raylet_socket_name=%s", rayConfig.rayletSocketName),
```

Comment:
I think the convention is to use double dashes for full names (`--raylet_socket_name`). why does gflag use single dash? Does gflag also support double?

Reply:
`gflags` supports double dashes too.